monai/bundle/utils.py

```diff
@@ -105,19 +105,43 @@
     "handlers_id": DEFAULT_HANDLERS_ID,
     "configs": {
         "tracking_uri": "$@output_dir + '/mlruns'",
+        "experiment_name": "default_experiment",
+        "run_name": "$None",
+        "not_rank0": (
```

Comment:
I would suggest "is_not_rank0", follows the style:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/config_item.py#L258
What do you think?

Thanks.